mne/minimum_norm/tests/test_inverse.py

```diff
@@ -797,6 +797,53 @@ def test_io_inverse_operator():
     _compare(inv_prep, inv_prep_prep)
 
 
+@testing.requires_testing_data
+@pytest.mark.parametrize('method', INVERSE_METHODS)
+@pytest.mark.parametrize('pick_ori', ['normal', None])
+def test_apply_inverse_cov(method, pick_ori):
+    """Test MNE with precomputed inverse operator on cov."""
+    if method == 'eLORETA':  # too slow
+        return
```

Comment:
use pytest.xfail?

Reply:
It seems like `xfail` should be reserved for times that we actually expect failures, not impose skips for time reasons. I'll just remove `eLORETA` from the list of tested methods, it seems cleaner